src/gui/config_tab.py

```diff
@@ -420,7 +420,7 @@ def keyPressEvent(self, event):
         if non_mod_key in modifiers_str:
             non_mod_key = ""
 
-        key_str = " + ".join(modifiers_str + [non_mod_key])
+        key_str = "+".join(modifiers_str + [non_mod_key])
```

Comment:
Was this bugget before with " + " or just cosmetic change?

Reply:
There was no bug, but this is just consistent with how we set our defaults and takes up a tiny amount less space when we print the table in the console that lists the hotkeys.

Reply:
OK